Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN apk -Uuv add --update --no-cache \
       less=530-r0 \
       libffi-dev=3.2.1-r4 \
       openssh-client=7.7_p1-r3 \
-      openssl=1.0.2o-r2 \
+      openssl=1.0.2p-r0 \
```

Comment:
This is needed because the build fails otherwise. We've had this a couple of times but Docker builds fail here if these are not on latest stable AFAIK.
I didn't want to invest much time to figure out why this is